src/pharmpy/tools/amd/run.py

```diff
@@ -54,7 +54,7 @@ def run_amd(
     occasion: Optional[str] = None,
     path: Optional[Union[str, Path]] = None,
     resume: Optional[bool] = False,
-    strictness: Optional[bool] = "minimization_successful or (rounding_errors and sigdigs>=0)",
+    strictness: Optional[str] = "minimization_successful or (rounding_errors and sigdigs>=0)",
```

Comment:
Does `strictness=None` mean no strictness?

Reply:
Yes. In that case no filtering is applied.